### ISSUE TITLE
A11y_AzureDataStudio_New deployment - SQL Server container image_Screen reader: Narrator is announcing the link 1 instead of label as Deployment pre requisites in left side

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18571]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info and Relationship;  #Narrator; #NVDA; #Reopened;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on Narrator
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment"
5. In the Deployment options, select "SQL Server container image" card.
6. Validate the required tools are installed and the "Required Tools" table shows the status as "Installed". Without the required tools the Select button will be disabled.
7. Click "Select".
8. Verify the Deployment pre requisites is reading by Narrator or not

### Note:
Similar issue can be observed in throughout the application wherever the steps to be followed in left side

### Actual Result:
Narrator is announcing the link 1 instead of label as Deployment pre requisites in left side   

### Expected Result:
Narrator should announce label as Deployment pre requisites 

### Attachment:
**Narrator**
![A11y_AzureDataStudio_New deployment - SQL Server container image](https://user-images.githubusercontent.com/100189407/155996119-04df8770-826e-467d-9dad-42e67b459e30.gif)
**NVDA**
![A11y_AzureDataStudio_New deployment - SQL Server container image](https://user-images.githubusercontent.com/100189407/155996369-f09d61c0-a478-4eaf-bf61-7032de938bea.png)

**Comment by alanrenmsft** [comment]
this is by design, the step number was not designed to have the step name, as soon as user are in the step, they will know the detail of it.

**Comment by VijayaBharathyA** [comment]
@alanrenmsft 
As discussed with Tanaz Narrator should announce Deployment pre requisites.

**Verified the issue in below environment :**
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Issue is partially fixed. NVDA is announcing name of the link . Narrator is not announcing.
![Narrator is announcing the link 1 instead of label as Deployment pre requisites in left side](https://user-images.githubusercontent.com/100189407/216549789-c1adb694-a18c-4821-a39b-79ff00a6f7bc.png)
![18571_No Repro with NVDA](https://user-images.githubusercontent.com/100189407/216549890-b6f088bc-4ef6-4912-8a97-c69503a14059.png)


**Note: Could you please reactivate the bug "Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

**Comment by alanrenmsft** [comment]
@VijayaBharathyA I've fixed the issue now, please use the latest insider build to verify: https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch

**Comment by VijayaBharathyA** [comment]
#closed;
### Verified the issue in below environment:
Version: 1.42.0-insider (user setup)
Commit: 385e4a22450d4549d58a262dcd503e636f1ed30d
Date: 2023-02-14T06:27:10.580Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Issue is fixed
![image](https://user-images.githubusercontent.com/100189407/218675293-6a7a894c-279e-4620-b8da-fb82badb3c30.png)
